pages/chatgroups.vue

```diff
@@ -55,8 +55,7 @@ export default {
   data() {
     return {
       chatLinks: {
-        Outsite: 'https://outsite.nl/#contact',
-        Announcements: 'https://chat.whatsapp.com/5yMBCohMukj1oCVR4lPHer',
```

Comment:
Interesting that the link is still in the Outsite chat though

Reply:
Hm yeah weird, but least the website is consistent now. It was a bit weird that you could just join here while you had to sign up and give your d.o.b. on the contact page before being able to join the announcement chat.